ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryMetricsFunctionalTest.java

```diff
@@ -48,13 +58,12 @@
 import org.junit.Test;
 import org.junit.rules.RuleChain;
 import org.junit.rules.Timeout;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 @Ignore
```

Comment:
The tests currently don't work on jenkins. There are two issues: 1) The metrics don't get recorded correctly within a single test 2) The metrics get clobbered between tests. There are no functional tests for metrics in the codebase so this seems like a bigger issue than just pull queries hence the tests are disabled for now

Reply:
That's a bummer, but I agree it makes sense to just skip it for now. Thanks, @vpapavas